l293x/l293x.go

```diff
@@ -0,0 +1,100 @@
+// Package l293x provides a driver to the L293/L293D H-bridge chip
+// typically used to control DC motors.
+//
+// Datasheet: https://www.ti.com/lit/ds/symlink/l293d.pdf
+//
+package l293x // import "tinygo.org/x/drivers/l293x"
+
+import (
+	"machine"
+)
+
+// Device is a motor without speed control.
+type Device struct {
+	d1, d2 machine.Pin
+	power  machine.Pin
+}
+
+// New returns a new Motor driver for GPIO-only operation.
+func New(d1, d2, p machine.Pin) Device {
+	return Device{
+		d1:    d1,
+		d2:    d2,
+		power: p,
+	}
+}
+
+// Configure configures the Device.
+func (d *Device) Configure() {
+	d.d1.Configure(machine.PinConfig{Mode: machine.PinOutput})
+	d.d2.Configure(machine.PinConfig{Mode: machine.PinOutput})
+	d.power.Configure(machine.PinConfig{Mode: machine.PinOutput})
+
+	d.Stop()
+}
+
+// Forward turns motor on in forward direction.
+func (d *Device) Forward() {
```

Comment:
What do you think about ClockWise / CounterClockWise instead of Forward / Backward? In a two opposite wheels configuration (a car for example) one wheel will go "backward" to move forward. I'm not a fan of CW/CCW either, but forward/backward sounds strange to me.

Reply:
I agree the directionality is probably relative to how to have motors mounted and the pins wired, so it is a bit arbitrary.

However, other motor control software tens to use the forward/backward nomenclature. 

For example https://github.com/adafruit/Adafruit-Motor-Shield-library/blob/master/examples/MotorTest/MotorTest.ino#L24